tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -28,16 +28,12 @@ public function testDiffReturnsDiff()
         $diff = <<<'TXT'
 --- Original
 +++ New
-@@ @@
+@@ -1,3 +1,4 @@
  <?php
  class Foo extends Bar {
 -    function __construct($foo, $bar) {
 +    public function __construct($foo, $bar)
 +    {
-         $this->foo = $foo;
```

Comment:
The current output builders provided by the `diff` package don't have options for this,
however I'm finishing up one that will allow this.